fem/fe_coll.hpp

```diff
@@ -1305,6 +1305,46 @@ class Local_FECollection : public FiniteElementCollection
    virtual int GetContType() const { return DISCONTINUOUS; }
 };
 
-}
+/// Radial basis function collection
+class KernelFECollection : public FiniteElementCollection
+{
+private:
+   int maxDim;
+   char d_name[32];
+   ScalarFiniteElement *Tr_Elements[Geometry::NumGeom];
+   ScalarFiniteElement *L2_Elements[Geometry::NumGeom];
+   int *SegDofOrd[2]; // for rotating segment dofs in 1D
+   int *OtherDofOrd;
+   bool ValidGeomType(Geometry::Type GeomType) const;
+
+public:
+   KernelFECollection(const int D, // dimension
+                      const int numPointsD, // number of points in each dimension, >= 2
+                      const int rbfType, // type of base radial basis function
+                      const int order, // order for reproducing kernels, -1 for none
+                      const double h, // smoothing parameter
+                      const double faceFactor =
+                         0.0, // 0.0 (points dx/2 from faces) to 1.0 (points on faces)
+                      const int intOrder = 2, // num integration points per 1d point
+                      const int distNorm = 2, // norm for distance calculation, usually 2
+                      const int mapType = FiniteElement::VALUE);
```

Comment:
This is the outward-facing interface to the RBF and RK finite elements. Where would be best to document these parameters?

Reply:
Right above the constructor declaration, with doxygen.

Reply:
Done.